frontend/configuration/setenv.sh

```diff
@@ -0,0 +1,26 @@
+# Copyright Siemens AG, 2019. Part of the SW360 Portal Project.
```

Comment:
we have used that file in testes, except that we did sue different mem settings, it seems to work well.